### PR TITLE
Fix/minor UI changes

### PR DESCRIPTION
**Pull request by henrikbossart**
## What does this pull request change?
Made some minor adjustments to a couple of components, nothing big.
Most notable change is the removal of the `RecipeSelector` component in favor of the EDS `Menu` component.

Other than that there is a bit more padding given to the YAML, a slight change to the height of the app, and a change in wording on a button.

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -19,12 +19,13 @@ function App() {
       style={{
         display: 'flex',
         padding: '20px',
-        overflow: 'auto',
+          height: '100vh'
```
Comment (ingeridhellen) [inline example/src/App.tsx:22]: The indentation seems off

File: packages/dm-core-plugins/src/header/HeaderPlugin.tsx [hunk 2]
```diff
@@ -28,7 +27,10 @@ const Icons = styled.div`
   }
 `
 
-const ClickableIcon = styled.div`
+const ClickableIcon = styled.button`
```
Comment (ingeridhellen) [inline packages/dm-core-plugins/src/header/HeaderPlugin.tsx:30]: I think this component if pre-EDS. The styling is wrong and it lacks a aria-label. We should instead use something like the TooltipButton I made.

File: packages/dm-core-plugins/src/header/HeaderPlugin.tsx [hunk 3]
```diff
@@ -99,31 +102,42 @@ export default (props: IUIPlugin): JSX.Element => {
...
-                setSelectedRecipe(getRecipeConfigAndPlugin(uiRecipeName))
-              }}
-            />
-          )}
+          <Menu open={appSelectorOpen} anchorEl={anchorEl}>
+            {recipeNames.map((recipe, index) => (
+              <Menu.Item
+                key={index}
+                onClick={() =>
+                  setSelectedRecipe(getRecipeConfigAndPlugin(recipe))
```
Comment (ingeridhellen) [inline packages/dm-core-plugins/src/header/HeaderPlugin.tsx:133]:
```suggestion
                  setSelectedRecipe(getRecipeConfigAndPlugin(recipe))
                  setAppSelectorOpen(false)
```
would be even better

File: packages/dm-core-plugins/src/header/HeaderPlugin.tsx [hunk 4]
```diff
@@ -99,31 +102,42 @@ export default (props: IUIPlugin): JSX.Element => {
...
+                }
+              >
+                {recipe}
+              </Menu.Item>
+            ))}
+          </Menu>
+          <h4 style={{ paddingLeft: 10 }}>{entity.label}</h4>
```
Comment (ingeridhellen) [inline packages/dm-core-plugins/src/header/HeaderPlugin.tsx:140]: We should try to replace every header element we find with EDS Typography

**Review by ingeridhellen — APPROVED** [review]
Nice change! Happy to see the RecipeSelector go.

Since the project is older than EDS, we have a lot of custom features that could have been solved better with EDS - like the RecipeSelector.

File: example/src/App.tsx (hunk 1 above)
Comment (henrikbossart) [reply example/src/App.tsx:22]: indeed... somehow my editor doesn't detect the config, and thus doesn't auto lint on save...... incredibly frustrating

File: packages/dm-core-plugins/src/header/HeaderPlugin.tsx (hunk 3 above)
Comment (henrikbossart) [reply packages/dm-core-plugins/src/header/HeaderPlugin.tsx:133]: indeed!

File: packages/dm-core-plugins/src/header/HeaderPlugin.tsx (hunk 4 above)
Comment (henrikbossart) [reply packages/dm-core-plugins/src/header/HeaderPlugin.tsx:141]: I suspect there are a couple of headings, so I'll open a new PR for that 👍

**Comment by henrikbossart** [comment]
FINALLY!

**Pull request merged**